commands/__tests__/account.test.ts

```diff
@@ -0,0 +1,66 @@
+import yargs, { Argv } from 'yargs';
+import * as list from '../account/list';
+import * as rename from '../account/rename';
+import * as use from '../account/use';
+import * as info from '../account/info';
+import * as remove from '../account/remove';
+import * as clean from '../account/clean';
+
+jest.mock('yargs');
+jest.mock('../account/list');
+jest.mock('../account/rename');
+jest.mock('../account/use');
+jest.mock('../account/info');
+jest.mock('../account/remove');
+jest.mock('../account/clean');
+jest.mock('../../lib/commonOpts');
+
+const commandSpy = jest
+  .spyOn(yargs as Argv, 'command')
+  .mockReturnValue(yargs as Argv);
+const demandCommandSpy = jest
+  .spyOn(yargs as Argv, 'demandCommand')
+  .mockReturnValue(yargs as Argv);
```

Comment:
Too bad that you have to do all this casting. Looked into the yargs types though and looks like its the only way.

Reply:
Yeah I spent a little time trying to avoid having to do this and this was still the best pattern I could find